recipes/configs/code_llama2/7B_lora_single_device.yaml

```diff
@@ -59,7 +59,7 @@ shuffle: True
 epochs: 1
 max_steps_per_epoch: null
 batch_size: 2
-gradient_accumulation_steps: 16
+gradient_accumulation_steps: 8  # Use to increase virtual batch size
```

Comment:
Sorry to be a pain but I feel like "effective batch size" is the more common term?

Reply:
:D